### PR TITLE
add "passenger" field to booking contact details

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
TODO: some tests probably

### CODE REVIEW

File: scripts/graphql-schema-snapshot.json [hunk 1]
```diff
@@ -7172,6 +7172,18 @@
             },
             "isDeprecated": false,
             "deprecationReason": null
+          },
+          {
+            "name": "passenger",
+            "description": null,
```
Comment (kantord) [inline scripts/graphql-schema-snapshot.json]: hmm, don't we need `description`?

Comment (jaroslav-kubicek) [reply scripts/graphql-schema-snapshot.json]: we should, just ain't no time :D

**Comment by michalsanger** [comment]
@kantord  I'm missing a test where is query with the added property and snapshot where I can see it resolves OK.

**Comment by jalbertsr** [comment]
query test with snapshot added @michalsanger quick check pls

**Pull request merged**